### PR TITLE
feat(Icons): logo icons

### PR DESCRIPTION
**Pull request by lisalupi**
## Type

- Feature

#### What is expected?

New category of icons in `@ultraviolet/icons`: `Logo`. 

To import them : 
`import { GoogleLogo } from '@ultraviolet/icons/logo'`

Come in 4 different sizes (small, medium, large and xlarge)

Capture d’écran 2024-09-13 à 10 20 16

### CODE REVIEW

File: packages/icons/src/components/Logo/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+`
+
+export type IconProps = {
+  size?: keyof typeof SIZES
+  className?: string
+  children: ReactNode
+}
+
+/**
+ * ProductIcon component is used to render a set of icons that are linked to a product or service.
```
Comment (matthprost) [inline packages/icons/src/components/Logo/Icon.tsx:28]: You can change the comment here according to the real component name 😄

File: packages/icons/src/components/Logo/__stories__/Sizes.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import type { StoryFn } from '@storybook/react'
+import { Stack, Text } from '@ultraviolet/ui'
+import { OutlookLogo } from '..'
+
+export const Sizes: StoryFn<typeof OutlookLogo> = props => (
+  <Stack gap={1}>
+    {(['small', 'medium', 'large', 'xlarge'] as const).map(size => (
+      <Stack direction="row" gap={1} alignItems="center">
```
Comment (matthprost) [inline packages/icons/src/components/Logo/__stories__/Sizes.stories.tsx:8]: a key is missing here

File: packages/icons/src/components/Logo/__stories__/List.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { Snippet, Stack, Text } from '@ultraviolet/ui'
+import * as ProductIcon from '..'
+
+export const List: StoryFn = props => (
+  <Stack gap={1}>
+    {Object.keys(ProductIcon).map(iconName => {
+      const FoundProductIcon = ProductIcon[iconName as keyof typeof ProductIcon]
+
+      return (
+        <Stack direction="row" gap={1} alignItems="center">
```
Comment (matthprost) [inline packages/icons/src/components/Logo/__stories__/List.stories.tsx:11]: a key is missing here too

File: scripts/generate-icons-file.ts [hunk 4]
```diff
@@ -20,6 +20,12 @@ const COMPONENTS = [
     input: 'packages/icons/src/components/CategoryIcon/assets',
     output: 'packages/icons/src/components/CategoryIcon/__generatedIcons__',
   },
+  {
+    name: 'Logo',
+    suffix: 'Logo',
+    input: 'packages/icons/src/components/Logo/assets',
+    output: 'packages/icons/src/components/Logo/__generatedIcons__',
+  },
```
Comment (matthprost) [inline scripts/generate-icons-file.ts:28]: Happy to see the script work that easily when adding new components 😄

File: packages/icons/src/components/Logo/assets/small/gmail.svg
Comment (matthprost) [inline packages/icons/src/components/Logo/assets/small/gmail.svg:1]: You can remove the small folder we don't need it the icon is an svg and there is no differences between the two

**Review by matthprost — CHANGES_REQUESTED** [review]

File: packages/icons/src/components/Logo/Icon.tsx (hunk 1 above)
Comment (lisalupi) [reply packages/icons/src/components/Logo/Icon.tsx:28]: oops

**Review by matthprost — APPROVED** [review]

**Review by fabienhebert — APPROVED** [review]

**Pull request merged**